english/eating-walls/find-a-toilet/magazine.md

```diff
@@ -0,0 +1 @@
+This is the time you get to spend reading something enjoyable. Makes time in the toilet enjoyable. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.